static/js/stat_var_hierarchy/stat_var_section_input.tsx

```diff
@@ -120,33 +142,69 @@ export class StatVarSectionInput extends React.Component<
     );
   }
 
-  private mouseMoveAction = (e) => {
-    let html = "This stat var has no data for any of the chosen places.";
-    if (!_.isEmpty(this.props.summary)) {
-      let availablePlaceTypes = Object.keys(
-        this.props.summary.placeTypeSummary
-      );
+  private getTooltipHtml(hasData: boolean): string {
+    let html = hasData
+      ? ""
+      : "This variable has no data for any of the chosen places.";
+    if (_.isEmpty(this.props.summary)) {
+      return html;
+    }
+    const availablePlaceTypes = Object.keys(
```

Comment:
consider pulling this out into a helper for readability

Reply:
done